commitizen-tools/commitizen
# feat: check if commit message is no more than 72 characters

## Description
Check if the length of commit message (the `prefix(scope): subject` part) does not exceed 72 characters.
Currently only supports ConventionalCommitsCz; JiraSmartCz and CustomizeCommitsCz not implemented.
An imperfect workaround for #191.

## Checklist


## Expected behavior

If run `cz commit` with `-cl` or `--check-length` argument, the commit would fail when `prefix(scope): subject` is longer than 72 chars.
Without the argument, the default behavior is to accept longer messages.

## Steps to Test This Pull Request

1. `cz commit -cl`
2. ENTER (type prefix: fix)
3. ENTER (empty scope)
4. 01234567890123456789012345678901234567890123456789012345678901234567 (for subject)
5. ENTER (empty body)
6. ENTER (not breaking change)
7. ENTER (no footer)

Then cz should exit with message `Length of commit message exceeded limit (73/72)` and error code 23.

## Additional context
